src/keep_actions_alive.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+from github import Github
+import os
+import pprint
+import requests
+
+
+def get_parameters():
+    # Add more options for adding arguments here,
+    # such as CLI args, AWS Secrets Manager, config file, etc.
+
+    # Get keyword arguments from environment variables
+    return {
+        "login_or_token": os.getenv(
+            "GH_LOGIN_OR_TOKEN"
+        ),
+        "organization": os.getenv("GH_ORGANIZATION"),
+        "user": os.getenv("GH_USER"),
```

Comment:
Comment so I don't forget from our call. I *believe* that all this `user` stuff can be scrapped as authenticating to GitHub should be fine with just the PAT.

Also using `requests.put()` to re-enable workflows should only need PAT too AFAIK.